docs/spec.md

```diff
@@ -2634,6 +2634,207 @@ planning to address this in
 
 &nbsp;[More Examples](https://github.com/openxla/stablehlo/tree/main/stablehlo/tests/interpret/dot_general.mlir)
 
+### dynamic_conv
+
+#### Semantics
+
+Computes dot products between windows of `lhs` and slices of `rhs` and produces
```

Comment:
You may just have this: 

This operation is functionally identical to
[convolution](https://github.com/openxla/stablehlo/blob/main/docs/spec.md#convolution)
op, with the `padding` specified dynamically as a value.